CHANGELOG.md

```diff
@@ -21,7 +21,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - The `"go.opentelemetry.io/otel/exporters/prometheus".New` now also returns an error indicating the failure to register the exporter with Prometheus. (#3239)
 - The prometheus exporter will no longer try to enumerate the metrics it will send to prometheus on startup.
    This fixes the `reader is not registered` warning currently emitted on startup. (#3291 #3342)
-- The `go.opentelemetry.io/otel/exporters/prometheus` exporter now correctly adds _total suffixes to counter metrics. (#3360)
+- The `go.opentelemetry.io/otel/exporters/prometheus` exporter now adds a unit suffix to metric names.
```

Comment:
Did you intend to remove this line?

Reply:
I did not intend to remove that line :/

Reply:
Fixed here: https://github.com/open-telemetry/opentelemetry-go/pull/3367